src/sync_template.sh

```diff
@@ -243,11 +273,16 @@ function pull_source_changes() {
 
   eval "git pull ${source_repo} --tags ${git_remote_pull_params}" || pull_has_issues=true
 
+  info "finished pulling from the source."
+  info "logging out from source ${SOURCE_REPO_HOSTNAME}."
+
   if [ "$pull_has_issues" == true ] ; then
     warn "There had been some git pull issues."
     warn "Maybe a merge issue."
     warn "We go on but it is likely that you need to fix merge issues within the created PR."
   fi
+
+  gh_login_target_github "${GITHUB_SERVER_URL}"
```

Comment:
Hi @alexvanderberkel just a small logical fix https://github.com/alexvanderberkel/actions-template-sync/pull/16

Think we are good to merge soon.
How are your tests? :rocket: